### ISSUE TITLE
t/column-headers.t failure in verbose mode with newer Test-Simple versions

### ISSUE THREAD

**Issue by ntyni** [Ovid/Test-Differences#15]
As reported also in https://github.com/Test-More/test-more/issues/810
we're seeing a failure in t/column-headers.t in (and only in!) verbose mode. This regressed with  https://github.com/Test-More/test-more/commit/b9c8b098fcb6be510d418b9356c3437b3660d67f ("Move diag newline to TAP formatter").

It's unclear where this should be fixed, but could you please have a look?

Thanks for your work.

```
#   Failed test 'got expected error output'
#   at t/column-headers.t line 14.
#          got: '#   Failed test 'both the same'
# #   at t/script/default-headers line 8.
# # +----+----------------+----------------+
# # | Elt|Got             |Expected        |
# # +----+----------------+----------------+
# # |   0|{               |{               |
# # *   1|  foo => 'bar'  |  foo => 'baz'  *
# # |   2|}               |}               |
# # +----+----------------+----------------+
# # Looks like you failed 1 test of 1.
# '
#     expected: '
# #   Failed test 'both the same'
# #   at t/script/default-headers line 8.
# # +----+----------------+----------------+
# # | Elt|Got             |Expected        |
# # +----+----------------+----------------+
# # |   0|{               |{               |
# # *   1|  foo => 'bar'  |  foo => 'baz'  *
# # |   2|}               |}               |
# # +----+----------------+----------------+
# # Looks like you failed 1 test of 1.
# '
#   Failed test 'got expected error output'
#   at t/column-headers.t line 36.
#          got: '#   Failed test 'both the same'
# #   at t/script/custom-headers line 8.
# # +----+----------------+----------------+
# # | Elt|Lard            |Chips           |
# # +----+----------------+----------------+
# # |   0|{               |{               |
# # *   1|  foo => 'bar'  |  foo => 'baz'  *
# # |   2|}               |}               |
# # +----+----------------+----------------+
# # Looks like you failed 1 test of 1.
# '
#     expected: '
# #   Failed test 'both the same'
# #   at t/script/custom-headers line 8.
# # +----+----------------+----------------+
# # | Elt|Lard            |Chips           |
# # +----+----------------+----------------+
# # |   0|{               |{               |
# # *   1|  foo => 'bar'  |  foo => 'baz'  *
# # |   2|}               |}               |
# # +----+----------------+----------------+
# # Looks like you failed 1 test of 1.
# '
# Looks like you failed 2 tests of 2.
t/column-headers.t .. 
not ok 1 - got expected error output
not ok 2 - got expected error output
1..2
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/2 subtests 

Test Summary Report
-------------------
t/column-headers.t (Wstat: 512 Tests: 2 Failed: 2)
  Failed tests:  1-2
  Non-zero exit status: 2
```

**Comment by DrHyde** [comment]
This explains the weirdness I've been seeing where `make test` passes but `perl -Mblib t/column-headers.t` fails!

**Comment by DrHyde** [comment]
Fixed in 570fcc1, and I'll push an update to the CPAN soon. Thanks!
